docker-compose.yml

```diff
@@ -30,11 +30,6 @@ services:
       - "9100:443"
     links:
       - api-server
-    volumes:
```

Comment:
These files are all copied into the container at build time, so we don't need also to mount them in as volumes